Answers.md

```diff
@@ -1 +1,18 @@
+1. Describe some of the differences between .forEach & .map.
+-There is a few key differences. The biggest being that .forEach doesn't have a return value (it's return value is undefined), whereas .map returns an array. So if you want to a transform the elements of an array, .map is what you need to use.
 
```

Comment:
The biggest takeaway from that should be:
forEach manipulates the array while map makes a copy.
there will be instances where you don't want to mess with the original